docs/instruments/ZJSONInstrument_v0.0.1.md

```diff
@@ -0,0 +1,1028 @@
+                    The number of columns in the table. 
+
+`options.rowSize` : Required int.
+                    The number of rows in the table.
+
+`options.order`: Required array of strings.
+                 An array of the names of `row` helpers, in order, that are to be formatted
+                 within this table.
+
+The size of this array must equal `options.rowSize`.
+
+At each index (or column) of every "row" element, the `helper` or `field` that is
+referenced inside that index has to have its `description` match. This description
+will be rendered as the column header if `options.showColHeaders` is true.
+
+The only exception is that a "static" helper will never have a corresponding header, 
+as its `description` is the static text itself. A column of static elements can have
+an empty column header. This is true for only static elements.
+
+`options.showColHeaders`: Boolean. Default is true.
+                          True or false that the table's column headers will be displayed as taken
+                          from each column's fields' matching `description`.
+
+`options.showDesc`: Boolean. Default is true.
+                    True or false that the table's `description` is displayed.
+
+`options.showAsFooter`: Boolean. Default is false.
+                        True or false that the description is displayed as a footer, instead of a
+                        header.
+
+### 4.1.5: Section
+
+A helper of type "section" is a vertical grouping of elements whose `rules` can affect its children's rules.
+
+```js
+{
+    "schema" : {
+        ...
+        "helpers" : {
+            "nameOfHelper" : {
+                "type": "section",
+                "description": {
+                    ...
+                },
+                "options": {
+                    "order": [
+                        string,
+                        string,
+                        ...
+                    ],
+                    "showDesc" : boolean
+                },
+                "rules": {
+                    ...
+                }
+        ...
+}
+```
+
+`options.order`: Required array of strings. An array of the names of `fields` or `helpers`, in order, that are to be
+                 formatted within this group. It may contain any field, but only helpers of the type "static".
+
+`options.showDesc`: Boolean. Default is true. True or false that the section's `description` is displayed as the header for
+                    section of elements.
+
+An implementation could choose to set this key to false, and display an independent "static" helper as a header instead.
+ 
+# 5.0: UI Properties
+
+The UI component of the Z-JSON Instrument contains information on how we render our `fields` and `helpers` objects
+on the front-end. It is composed of JSON objects whose keys match those of the field or helper object it's describing.
+This component should only make decisions on which HTML or HTML Form element those objects get rendered as,
```

Comment:
What if it's not rendered to HTML?

Reply:
we define an other ui object

Reply:
I think this object is meant to be the mapping from field to HTML object... did you have something else in mind ?

Reply:
mobile app, desktop app, some theoretically successor to HTML.. if it's only for mapping to HTML, why not just use HTML directly?

Reply:
i see, i guess it doesn't necessarily have to be on the browser